client/analytics/report/orders/store/resolvers.js

```diff
@@ -0,0 +1,21 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import apiRequest from '@wordpress/api-request';
+
+export default {
+	async getOrders() {
+		try {
+			const orders = await apiRequest( { path: '/wc/v3/orders' } );
```

Comment:
This will also need to handle query args correct?

Reply:
Right, it will need to handle those as well. @coderkevin is looking into incorporating some features of fresh-data into wp.data, hopefully including using query args as keys for retrieving already fetched data. I think its a good idea to revisit this as his efforts are further along